compiler/dyno/include/chpl/util/terminal.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHPL_UTIL_TERMINAL_H
+#define CHPL_UTIL_TERMINAL_H
+
+#include <string>
+
+namespace chpl {
+
+enum TermColorName {
+    clear,
+    black,
+    red,
```

Comment:
Any reason why these aren't `SCREAM_AT_ME` cased? And maybe put them in a namespace or something e.g., `TermColor::CLEAR`